superset/contrib/docker/docker-entrypoint.sh

```diff
@@ -27,8 +40,8 @@ elif [ "$SUPERSET_ENV" = "development" ]; then
     celery worker --app=superset.sql_lab:celery_app --pool=gevent -Ofair &
     # needed by superset runserver
     (cd superset/assets/ && npm ci)
-    (cd superset/assets/ && npm run dev) &
-    FLASK_ENV=development FLASK_APP=superset:app flask run -p 8088 --with-threads --reload --debugger --host=0.0.0.0
+    (cd superset/assets/ && npm run dev-server -- --host=0.0.0.0 --port=8088 --supersetPort=8081) &
```

Comment:
Could you please add a comment or explain how the ports here are being used? In particular port `8081`.

Reply:
It's explained in their docs [apache/incubator-superset/CONTRIBUTING.md#building](https://github.com/apache/incubator-superset/blob/release--0.32/CONTRIBUTING.md#building) and it might be too much to explain this wiring... but let's try; wdyt about 885dad278550dda514bdfe68e74f6b59fd3e2f01?